diagram-viewer/src/main/java/com/powsybl/diagram/viewer/sld/SingleLineDiagramViewController.java

```diff
@@ -360,11 +369,20 @@ public void updateFrom(final ObjectProperty<Network> networkProperty) {
         highlightStyleProviderCheckBox.disableProperty().bind(Bindings.createBooleanBinding(() -> networkProperty.get() != null, networkProperty).not());
         topologicalStyleProviderCheckBox.disableProperty().unbind();
         topologicalStyleProviderCheckBox.disableProperty().bind(Bindings.createBooleanBinding(() -> networkProperty.get() != null, networkProperty).not());
-
+        // Hide SMART & CGMES if no network available
+        if (networkProperty.get() == null) {
```

Comment:
This is not straightforward, isn't it easier to be robust to a null network when SMART or CGMES are selected?